packages/rrweb/src/replay/index.ts

```diff
@@ -2179,7 +2179,7 @@ export class Replayer {
   private drawMouseTail(position: { x: number; y: number }, pointerId: number) {
     const pointer = this.pointers[pointerId];
```

Comment:
This might be the trick to tell typescript that `pointer` could be null, then we'll get all the typehints:

```suggestion
    const pointer = this.pointers.at(pointerId);
```

Reply:
that gives me a lint error :(
```
Unsafe assignment of an `any` value. 

```

eslint[@typescript-eslint/no-unsafe-assignment](https://typescript-eslint.io/rules/no-unsafe-assignment)

Reply:
ooh, think `pointers` is a Record type, i assumed array without looking. 

nbd.